src/betterproto/compile/naming.py

```diff
@@ -11,3 +13,19 @@ def pythonize_field_name(name: str) -> str:
 
 def pythonize_method_name(name: str) -> str:
     return casing.safe_snake_case(name)
+
+
+def pythonize_enum_member_names(names: Iterable[str]) -> List[str]:
+    """Removes any of the same characters from an Enum member's names."""
```

Comment:
I'm not sure this is the quite right thing to do. Surely what we really want is to specifically check if all the enum names are prefixed with exactly the capitalised snake case name of the enum (as per the convention), and if so then remove it.

Removing ANY common prefix could be quite surprising... especially if the enum only has one value ;)

Reply:
I'll revert this then